cmd/cloudtrail/write-events_test.go

```diff
@@ -0,0 +1,79 @@
+package cloudtrail
+
+import (
+	"testing"
+
+	"github.com/aws/aws-sdk-go-v2/service/cloudtrail"
+	"github.com/aws/aws-sdk-go-v2/service/cloudtrail/types"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestLoadConfiguration(t *testing.T) {
+
+	config, err := LoadConfiguration()
+
+	if err != nil {
+
+		t.Errorf("Error loading configuration: %v", err)
+	}
+
+	if config == nil {
+		t.Error("Configuration is nil")
+	}
+
+}
+
+func TestFilterUsers(t *testing.T) {
```

Comment:
Generally for the tests:
- add more test cases, split them into different tests, don't keep them in a single big one.
- test odd cases e.g. the filter is empty
- test the different flags you pass to `FilterUsers`